### ISSUE TITLE
generic set function

### ISSUE THREAD

**Issue by theova** [AuditeMarlow/base16-manager#31]
Hello!

As `base16-manager` provides a quite good infrastructure to set themes, I think it would be a good idea to support more (or all) templates of [base16](https://github.com/chriskempson/base16).

Therefore I would like to work on a generic set function. The main goal is to simplify the support of new templates and minimize the code, so that we don't have to write a new `set()` function for each template.

A first idea is the following pseudo-code:
```
set_generic() {
  local package=$1
  local theme=$2
  local cmnt_string=$3
  local assign_string=$4
  local config=$5
  local file="${CONFIG_PATH}/${package}/themes/base16-${theme}"

  for line in $file
    delete_lines_containing_left_side_of_assignment($config)
  end
  delte_obsolete_comments($config)

  cat $file >> $config # append new colors.
}
```

What do you think?

**Comment by AuditeMarlow** [comment]
I think this is a great idea and definitely something that should be integrated in base16-manager.

**Comment by AuditeMarlow** [comment]
Merged #32.
